### PR TITLE
dev-java/jide-oss: bump to 3.7.12

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: dev-java/jide-oss/jide-oss-3.7.12.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+DESCRIPTION="JIDE Common Layer (Professional Swing Components)"
+HOMEPAGE="https://github.com/jidesoft/jide-oss"
+SRC_URI="https://github.com/jidesoft/jide-oss/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2-with-exceptions"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="virtual/jdk:1.8"
```
Comment (fordfrog) [inline dev-java/jide-oss/jide-oss-3.7.12.ebuild:23]: please comment here why it needs to be restricted to 1.8

File: dev-java/jide-oss/jide-oss-3.7.12.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+S="${WORKDIR}/${PN}-${MY_COMMIT}"
+
+JAVA_GENTOO_CLASSPATH_EXTRA="libs/ui.jar"
+JAVA_SRC_DIR=( "src" "src-jdk8" )
+JAVA_RESOURCE_DIRS=( "src" "properties" )
+
+src_prepare() {
+	default
+	rm libs/junit-4.10.jar || die
```
Comment (fordfrog) [inline dev-java/jide-oss/jide-oss-3.7.12.ebuild:37]: tests are implemented but don't work?

Comment (vaukai) [reply dev-java/jide-oss/jide-oss-3.7.12.ebuild:37]: Haven't yet tried the tests.

**Comment by fordfrog** [comment]
@vaukai nice, thanks! i also added comment to the failing test at 64778250841a37a85fa280a5008e1f7614a193ec

**Pull request closed without merging**